### PR TITLE
[6.5.x] DROOLS-2446 : Concurrent spreadsheet table validation doesn't work properly

### PR DESCRIPTION
**Pull request by Rikkola**
https://issues.jboss.org/browse/DROOLS-2446

### CODE REVIEW

File: kie-wb-common-services/kie-wb-common-services-backend/src/main/java/org/kie/workbench/common/services/backend/validation/asset/ValidatorBuildService.java [hunk 1]
```diff
@@ -60,7 +60,7 @@ public ValidatorBuildService( final @Named("ioStrategy") IOService ioService,
         this.projectService = projectService;
     }
 
-    public List<ValidationMessage> validate( final Path resourcePath,
+    public synchronized List<ValidationMessage> validate( final Path resourcePath,
                                              final String content ) {
```
Comment (jomarko) [inline kie-wb-common-services/kie-wb-common-services-backend/src/main/java/org/kie/workbench/common/services/backend/validation/asset/ValidatorBuildService.java]: Not sure if we use some code style for 6.5.x, however could we indent method parameters?

Comment (Rikkola) [reply kie-wb-common-services/kie-wb-common-services-backend/src/main/java/org/kie/workbench/common/services/backend/validation/asset/ValidatorBuildService.java]: Sure. Fixed this already for the other branches.

**Comment by Rikkola** [comment]
jenkins please retest this

**Comment by Rikkola** [comment]
Has acks and is ready to go in.

**Pull request merged**